scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -200,7 +200,7 @@ abstract class LocalScanner(
     }
 
     override suspend fun scanPackages(
-        packages: Collection<Package>,
+        packages: Set<Package>,
```

Comment:
Is `Package` the right key to de-duplicate it. Or should it be `distinct by` the `Identifier` ?

Reply:
I guess we *could* more aggressively de-duplicate based on the `id`, but actually we should never have different `Package`s with the same `id` here, so I believe we are fine.